content/_data/upgrades/2-190-1.adoc

```diff
@@ -1 +1,10 @@
-No notable changes requiring upgrade notes.
+==== Jenkins no longer creates symbolic links inside project or build directories
+
+https://issues.jenkins-ci.org/browse/JENKINS-37862[JENKINS-37862]
+
+There were a lot of bugs and limitations with use of symbolic links for build directories, especially on platforms like Windows.
```

Comment:
Can we provide some references here? Do we want to?

Reply:
There is a JIRA link with more context, should be enough